README.md

```diff
@@ -137,6 +142,24 @@ export INDEXER_SHA=""
 export INDEXER_DISABLED=""
 ```
 
+## Debugging for teal developers
+
+For detailed information on how to debug smart contracts and use tealdbg CLI,please consult with [Algorand Development Portal :: Smart Contract Debugging](https://developer.algorand.org/docs/features/asc1/debugging/#setting-the-debugger-context).
+
+Algorand smart contract debugging process uses `tealdbg` command line of algod instance(algod container in sandbox).
+
```

Comment:
Please add the --listen flag comments in this area.

Reply:
Right away Sir!

Reply:
It is done!